documentation/modules/auxiliary/scanner/http/zabbix_login.md

```diff
@@ -0,0 +1,48 @@
+## Vulnerable Application
+
+This module attempts to login to zabbix server, by default module will use default credentials and also supports user defined login credentials
```

Comment:
This sentence could do with some English improvements. I'll try edit this up.

Reply:
Also what are the default credentials for Zabbix? Might be good to reference them here.

Reply:
zabbix default creds are Admin:zabbix

Reply:
Would you mind explaining this here, perhaps just putting in as `(Admin:zabbix)`? Just makes it easier for readers to reference who might not be familiar with the default login credentials for Zabbix.

Reply:
Resolved this with https://github.com/rapid7/metasploit-framework/pull/14297/commits/bd578324941b2ef1f65b452c80c45875f9bf9b68